src/libasr/compiler_tester/tester.py

```diff
@@ -64,11 +64,11 @@ class fg:
 
 
 def color(value):
-    return "\033[" + str(int(value)) + "m"
+    return ""
```

Comment:
Why we have the function if we are just going to return an empty string? Can we just replace, `color(...)` calls with "" instead? Or may be for Windows return an empty string but for other operating systems return `"\033[" + str(int(value)) + "m"`.

Reply:
Yup, this was used for debugging purposes.
If this method is valid for fixing, then I thought to add an option `--no-color` to print the output without color.

Reply:
> I thought to add an option `--no-color` to print the output without color.

I think its already there? I have used it many times before.

Reply:
Do you mean, for run_tests?

Reply:
https://github.com/lcompilers/lpython/blob/fa026e1437e2e40e51b302683364c02e603c9d09/src/bin/lpython.cpp#L1409

Reply:
Yup, that is used for AST and ASR. But, I was thinking to add new option in run_tests. Actually, I had problems with run_tests output color in Windows and here in the CI I had the problem with `check` symbol. So, adding new option would be right fixing this.